Win/ObjFileManip.cpp

```diff
@@ -13716,17 +13716,35 @@ static int writeOBJMtlFile()
                 && !( gOptions->pEFD->chkLeavesSolid && (gBlockDefinitions[type].flags & BLF_LEAF_PART) ) )
             {
                 // cutouts or alpha
-                gModel.usesRGBA = 1;
-                gModel.usesAlpha = 1;
-                typeTextureFileName = textureRGBA;
-                sprintf_s(mapdString,256,"map_d %s\n", textureAlpha );
+                if ((gOptions->exportFlags & EXPT_SKFB))
+                {
+                    gModel.usesRGBA = 1;
+                    gModel.usesAlpha = 0;
```

Comment:
RGBA texture is enough for our use (RGB for diffuse, and set it in Transparency channel with Alpha instead or Luminance). It would be redundant to also export alpha texture (luminance + alpha values)